xBRZTester/Program.cs

```diff
@@ -8,22 +10,25 @@ internal class Program
     {
         private static void Main(string[] args)
         {
-            SaveScaledImage();
+            if (args.Length != 4)
+            {
+                Console.WriteLine("Usage: xBRZTester.exe ScaleFactor(2-6) InputPath OutputPath");
+                Console.WriteLine("\tEx:\txBRZTester.exe 4 C:\\example\\input.png output.png");
+                Console.WriteLine("\nPress any key to exit");
+                Console.ReadKey();
+
+            }
+            else
+            {
+                SaveScaledImage(Int32.Parse(args[0]), args[1], args[2]);
+            }
```

Comment:
I generally don't use if-else statements in my code. Instead, I suggest a guard clause.

```suggestion
            if (args.Length != 4)
            {
                Console.WriteLine("Usage: xBRZTester.exe ScaleFactor(2-6) InputPath OutputPath");
                Console.WriteLine("\tEx:\txBRZTester.exe 4 C:\\example\\input.png output.png");
                Console.WriteLine("\nPress any key to exit");
                Console.ReadKey();
                return;
            }
            
            SaveScaledImage(Int32.Parse(args[0]), args[1], args[2]);
```